doc(release) Release notes for modules 24.04 on prem (#3255)
centreon/centreon-documentation: versioned_docs/version-24.04/releases/centreon-commercial-extensions.mdx

If you have feature requests or want to report a bug, please contact support.

> Read more about version 24.04 in our [blog post](https://www.centreon.com/centreon-announces-its-latest-software-release-centreon-24-04/).

<ExpandCollapseAll />

## Centreon MAP

### 24.04.0

Release date: `April 30, 2024`

<details open>
  <summary>Enhancements</summary>

> - To facilitate smooth transition from MAP Legacy to MAP, we have made sure that features and enhancements already available in previous versions are fully compatible.

Reviewer: ```suggestion
- To facilitate a smooth transition from MAP Legacy to MAP, we have made sure that features and enhancements already available in previous versions are fully compatible.
```

</details>

## Centreon BAM

### 24.04.0

Release date: `April 30, 2024`

<details open>
  <summary>Enhancements</summary>

- [API] Added an endpoint to GET Business Activities.

</details>

## Centreon MBI

### 24.04.0

Release date: `April 30, 2024`

<details>
  <summary>Compatibility</summary>

- Compatibility with other 24.04 components.

</details>

## Centreon Auto Discovery

### 24.04.0

Release date: `April 30, 2024`

<details>
  <summary>Compatibility</summary>

- Compatibility with other 24.04 components.

</details>

## Centreon Monitoring Connectors Manager (formerly Plugin Packs Manager)

### 24.04.0

Release date: `April 30, 2024`

<details>
  <summary>Compatibility</summary>

- Compatibility with other 24.04 components.

</details>

## Centreon License Manager

### 24.04.0

Release date: `April 30, 2024`

<details>
  <summary>Compatibility</summary>

- Compatibility with other 24.04 components.

</details>

## Centreon Anomaly Detection

### 24.04.0

Release date: `April 30, 2024`

<details open>
  <summary>Enhancements</summary>

> - [API] Added an endpoint to UPDATE Cloud Token.

Reviewer: ```suggestion
- [API] Added an endpoint to UPDATE Cloud tokens.
```

- [API] Added an endpoint to GET Business Activities.

</details>

## Centreon IT Edition Extensions

### 24.04.0

Release date: `April 30, 2024`

<details open>
  <summary>Enhancements</summary>

> - [API] Added a new endpoint to list shares of playlists.
> - [API] Added a new endpoint to share playlists.
> - [API] Added a new endpoint to create a playlist of dashboards.
> - [API] Added a new endpoint to list playlists of dashboards.
> - [API] Added a new endpoint to revoke the share of a contact group on a playlist.

Reviewer: ```suggestion
New playlist feature: dashboard playlists allow you to display several dashboards sequentially without any interaction. Ideal for broadcast on a TV screen. 
	
You can now:
- create dashboard playlists.
- display the list of dashboard playlists.
- share playlists.
- display the list of playlist shares.
- revoke the sharing of a playlist to a contact group.
```

If you have feature requests or want to report a bug, please contact support.

> Read more about version 24.04 in our [blog post](https://www.centreon.com/centreon-announces-its-latest-software-release-centreon-24-04/).

<ExpandCollapseAll />

## Centreon MAP

### 24.04.0

Release date: `April 30, 2024`

<details open>
  <summary>Enhancements</summary>

- To facilitate a smooth transition from MAP Legacy to MAP, we have made sure that features and enhancements already available in previous versions are fully compatible.

</details>

## Centreon BAM

### 24.04.0

Release date: `April 30, 2024`

<details open>
  <summary>Enhancements</summary>

- [API] Added an endpoint to GET Business Activities.

</details>

## Centreon MBI

### 24.04.0

Release date: `April 30, 2024`

<details>
  <summary>Compatibility</summary>

- Compatibility with other 24.04 components.

</details>

## Centreon Auto Discovery

### 24.04.0

Release date: `April 30, 2024`

<details>
  <summary>Compatibility</summary>

- Compatibility with other 24.04 components.

</details>

> ## Centreon Monitoring Connectors Manager (formerly Plugin Packs Manager)

Reviewer 2: ```suggestion
## Centreon Monitoring Connectors Manager
```

### 24.04.0

Release date: `April 30, 2024`

<details>
  <summary>Compatibility</summary>

- Compatibility with other 24.04 components.

</details>

## Centreon License Manager

### 24.04.0

Release date: `April 30, 2024`

<details>
  <summary>Compatibility</summary>

- Compatibility with other 24.04 components.

</details>

## Centreon Anomaly Detection

### 24.04.0

Release date: `April 30, 2024`

<details open>
  <summary>Enhancements</summary>

- [API] Added an endpoint to UPDATE Cloud tokens.

> - [API] Added an endpoint to GET Business Activities.

Reviewer 2: ```suggestion
```
